versionwarning.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+    // adapted 2020-05 from https://scikit-learn.org/versionwarning.js
+    const latestStable = '0.20';
+    const goodPaths = ['stable', 'dev', latestStable];
+    const showWarning = (msg) => {
+        $('.body[role=main]').prepend(`<div class="d-block devbar alert alert-danger">${msg}</div>`)
```

Comment:
for the formatting to work right, I actually want to place this at a different point in the tree...  looks like selecting `div id='navbar'` and using `.after()` instead of `prepend` should do the trick.  @hoechenberger @larsoner do you know how I should change `.body[role=main]` to properly select the navbar?  I haven't really used JS in a long time.

Reply:
fixed, w/ @larsoner 's help.  I committed the fix directly rather than do another PR